lib/hammer_cli_katello/output/fields.rb

```diff
@@ -10,6 +10,9 @@ class ChecksumFilePair < ::Fields::Field
       class Dependency < ::Fields::Field
       end
 
+      class Array < ::Fields::Field
```

Comment:
Yeah, I was thinking similar. Keeping list and adding an optional ``separator`` argument. Or possibly extending ``List`` with ``VerticalList`` only changing the separator that ``List`` uses?